tests/test_vector_db_service.py

```
@@ -1,31 +1,59 @@
         return {'matches': [{'id': 'id1', 'score': 0.99}]}
 
 class MockPinecone:
     def __init__(self, api_key):
         self.api_key = api_key
+        self.describe_index_calls = 0
+        self.create_index_calls = 0
+        self.error_on_describe_index_calls = []
+        self.error_on_create_index_calls = []
+        self._index_instance = None
 
     def Index(self, name):
```

Ruff (0.14.1)

Unused method argument: `name`

(ARG002)